pkg/cli/alpha/internal/generate.go

```diff
@@ -354,15 +354,15 @@ func createWebhook(resource resource.Resource) error {
 // Gets flags for webhook creation.
 func getWebhookResourceFlags(resource resource.Resource) []string {
 	var args []string
-	if resource.HasConversionWebhook() {
-		args = append(args, "--conversion")
-	}
 	if resource.HasValidationWebhook() {
 		args = append(args, "--programmatic-validation")
 	}
 	if resource.HasDefaultingWebhook() {
 		args = append(args, "--defaulting")
 	}
+	if resource.HasConversionWebhook() {
+		args = append(args, "--conversion", "--spoke", resource.Webhooks.Spoke)
```

Comment:
Probably add validation that `--conversion` needs to be set, to have `--spoke` version? Wondering if we need `--conversion` in that case? Can it be automatically considered true if user specifies `--spoke`?

Reply:
Actually, rethinking about this - iiuc it is:
1. if `conversion` is specified - the existing functionality remains.
2. if `spoke` is specified - then `Hub` and `Spoke` interfaces are scaffolded.

In that case would it make sense to validate that `--conversion` is set if `--spoke` is specified.

Reply:
If we do not inform the spoke that will fails because we will need to have that right.
Without the spoke we cannot implement it